assets/packages/adminbasics/src/components/confirmdeletemodal.js

```diff
@@ -3,7 +3,7 @@ const ConfirmDeleteModal = function (options) {
 
     options.fnOnShown = options.fnOnShown || function () {};
     options.fnOnHide = options.fnOnHide || function () {};
-    options.removeOnClose = options.removeOnClose || function () {};
+    options.removeOnClose = options.removeOnClose || true;
```

Comment:
Hm, isn't `removeOnClose` a function? How does that work with `true`?

Reply:
Is is used as boolean throughout the code.
![image](https://user-images.githubusercontent.com/4942495/135085969-12a788fc-8976-4378-bfed-8fe146f81a82.png)

Reply:
OK :)